docs/api/source/index.rst

```diff
@@ -13,7 +13,24 @@ For Solid Servers that support notifications, they are sent when a resource is
 created, updated, deleted, or when a child resource is added to or removed from
 a container.
 
-**This library currently only supports notifications sent via WebSockets.**
+.. admonition:: Supported Protocols
+   :class: caution
+
+   This library currently only supports an `early version of WebSocket
+   notifications
+   <https://docs.inrupt.com/ess/1.1/services/service-websocket/>`__, which
+   formed the basis of the Solid Notification Protocol 1.0 draft specification.
+
+   The `Solid Notification Protocol 1.0
```

Comment:
I'm not quite sure what this paragraph is adding, since this library doesn't support the draft spec.  
Aside:  as a general rule of thumb, we try to avoid forward-looking statements in our docs. 

If you want to say that it's subject to change, that's okay.

Reply:
Yeah, that's the intent: there's a newer protocol, we'll be hopefully implementing it soon, but we don't currently implement it.

Reply:
Please lets avoid making any roadmap statements in here.